extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -294,15 +295,21 @@ export class ShareWidget {
   loadProviders() {
     this.loadRequiredExtensions();
 
-    this.requestService_.loadShareConfig().then((config) => {
-      const providers =
-        config &&
-        (config[SHARE_PROVIDERS_KEY] || config[DEPRECATED_SHARE_PROVIDERS_KEY]);
-      if (!providers) {
-        return;
-      }
-      this.setProviders_(providers);
-    });
+    const shareEl = this.storyEl.querySelector(
+      'amp-story-social-share, amp-story-bookend'
+    );
+    once(
```

Comment:
This does not work, if you don't call the returned function, you run the passed callback each time. You can check the previous implementation from request-service for reference.

Reply:
yes, I have refactored it back to use once in the request-service file.